plugins/kubernetes-backend/api-report.md

```diff
@@ -191,9 +198,16 @@ export interface KubernetesFetcher {
 //
 // @public (undocumented)
 export interface KubernetesObjectsProvider {
+  // (undocumented)
+  getCustomResourcesByEntity(
+    entity: Entity,
```

Comment:
Does it make sense to maybe wrap all these up in an object instead of adding more and more params in the future?

Reply:
will do